pkg/server/image_pull_test.go

```diff
@@ -288,3 +288,25 @@ func TestDefaultScheme(t *testing.T) {
 		assert.Equal(t, test.expected, got)
 	}
 }
+
+func TestEncryptedImagePullOpts(t *testing.T) {
+	for desc, test := range map[string]struct {
+		keyModel     string
+		expectedOpts int
+	}{
+		"node key model should return one unpack opt": {
+			keyModel:     criconfig.EncryptionKeyModelNode,
+			expectedOpts: 1,
+		},
+		"no key model selected should default to node key model": {
```

Comment:
ah default to key model... hmm.. Maybe it should be more explicit give this is a security related switch.